doc/rst/language/spec/modules.rst

```diff
@@ -573,50 +573,16 @@ enumerated type (unless the module has been renamed to ``_``, as described
 earlier). It is an error to provide a name in a ``limitation-clause`` that does
 not exist or is not visible in the respective module or enumerated type.
 
-If a type or type's secondary methods are defined in the used module, then any
-instances of the type obtained in the scope of the use may access the fields and
-methods of that type, regardless of the ``limitation-clause``. These fields
-and methods cannot be specified in a ``limitation-clause`` on their own.  The
-privacy of use statements is also ignored when determining if an instance can
-access the fields and methods, for similar reasons.
+If an ``only`` list is left empty or an ``except`` list is followed by :math:`*`
+then no symbols are made available to the scope without prefix.
```

Comment:
Let us add precision by rewording as "... or an except list contains `*` ..."

Reply:
I think "contains" is too general.  `except a, *;` is not allowed.  Just `except *;` is.

Reply:
I agree. Still, "except list is followed by `*`" is not correct either, because it means something like `except A, B, *`

What is a good way to phrase it correctly? Borrowing from the same file: `except is followed by *`